conda_env/env.py

```diff
@@ -47,9 +48,10 @@ def validate_keys(data, kwargs):
         print('')
 
     deps = data.get('dependencies', [])
+    depsplit = re.compile(r"[\s=]")
```

Comment:
would it make sense to handle >, <, and possible ~ also in this regex?

Reply:
(hopefully) implemented in 2b032b6